data/migrations/20190112093745_project_table.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('projects', tbl => {
+      tbl.increments();
+    //   tbl.integer().notNullable().unsigned();
+      tbl.string('project_name', 255).unique();
+      tbl.string('project_description', 255);
+      tbl.boolean('project_completed').notNullable().defaultTo(false);
+  })
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.dropTableIfExists('project');
```

Comment:
You're missing an 's' here! As said, use a variable so that you'll have intellisense (assuming you're using a text editor with that) and so that it'll be consistent. For example:
```javascript
const projects = 'projects';

exports.up = function(knex, Promise) {
  return knex.schema.createTable(projects, tbl => {
      tbl.increments();
      tbl.string('project_name', 255).unique();
      tbl.string('project_description', 255);
      tbl.boolean('project_completed').notNullable().defaultTo(false);
  })
};

exports.down = function(knex, Promise) {
  return knex.schema.dropTableIfExists(projects);
};
```